spec.bs

```diff
@@ -666,6 +680,9 @@ scope</dfn> given a [=pre-specified report parameters=] |params| and a
     filtering ID max bytes=].
 1. [=Assert=]: |filteringIdMaxBytes| is [=set/contained=] in the [=valid
     filtering ID max bytes range=]
+1. Let |requestedMaxContributions| be |params|' [=pre-specified report
+    parameters/requested max contributions=].
+1. [=Assert=]: |requestedMaxContributions| is null or greater than zero.
```

Comment:
It might be nice to clarify what happens when the caller asks for zero contributions. I think it should fall back to the default size.

Reply:
This would require a little refactor though. Currently, we say "a positive integer" all over the place, so I think that the caller is effectively burdened with validating positivity.